spec/run_cd4pe_job_spec.rb

```diff
@@ -136,6 +136,13 @@
   
       expect(job_helper.docker_run_args).to eq("#{arg1} #{arg2} #{arg3}")
     end
+
+    it 'Sets the HOME and REPO_DIR env vars' do
+      job_helper = CD4PEJobRunner.new(working_dir: @working_dir, job_token: @job_token, web_ui_endpoint: @web_ui_endpoint, job_owner: @job_owner, job_instance_id: @job_instance_id, logger: @logger)
```

Comment:
I think we want to actually test this from a script that was executed by `CD4PEJobRunner`, and we'd want to test the values as seen from the script, not from this test code